File: Kattis/C++/Judging-Moose/Judging-Moose.cpp
#include <bits/stdc++.h>

using namespace std;
using i64 = long long;
using d6 = long double;

int main()
{
    i64 l, r, x;
    cin >> l >> r;

    if(l > r)
    {
        x = l * 2;
        cout << "Odd " << x << endl;
    }
    if(r > l)
    {
        x = r * 2;
        cout << "Odd " << x << endl;
    }
    if(l == r && l != 0 && r != 0)
    {
        x = r * 2;
        cout << "Even " << x << endl;
    }
    if(l == 0 && r == 0)
    {
        cout << "Not a moose" << endl;
    }
}

File: Kattis/Python/Judging-Moose.py

def main():
    l, r = input().split()
    l = int(l)
    r = int(r)

    if(l > r):
        x = l * 2
        print("Odd", x, end="\n")
    if(r > l):
        x = r * 2
        print("Odd", x, end="\n")
    if(l == r and l != 0 and r != 0):
        x = r * 2
        print("Even", x, end="\n")
    if(l == 0 and r == 0):
        print("Not a moose\n")

if __name__ == "__main__":
    main()